README.md

```diff
@@ -196,7 +196,6 @@ this:
 ```cpp
 int main(int argc, char** argv) {
     CLI::App app{"App description"};
-    argv = app.ensure_utf8(argv);
```

Comment:
Why is this removed? This is by far the best way to do this - `parse()` / `parse_from_cli_args()` does black magic on some operating systems, while this reliably works everywhere.

Reply:
In this section of the documentation it is covering the most basic application,  in a vast majority of applications this is not necessary.  I can add it back if you think it valuable, but to me it creates a lot of potentially unnecessary questions when just getting started.